### PR TITLE
Download data v20

### PR DESCRIPTION
**Pull request by kgaonkar6**
### Purpose/implementation Section
v20 data release
- Changes:
 - Update copy_number_consensus_call with manta calls with FILTER==PASS
    - pbta-cnv-consensus.seg.gz
 - Update focal-cn-file-preparation with deep deletion status and with updated pbta-cnv-consensus.seg.gz above
    - consensus_seg_annotated_cn_autosomes.tsv.gz
    - consensus_seg_annotated_cn_x_and_y.tsv.gz
 - Add cnv and indel hostspots from MSKCC and TERT promoters  
    - pbta-snv-scavenged-hotspots.maf.tsv.gz
 - Update histology information 
   - latest pathology diagnosis for 7316-2686 
   - BS_0TCRV9AC as sample_type == Normal

### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [x] Is doc/data-format.md up to date?
- [x] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date?
- [x] Was download-data.sh tested and did it complete without error?
-->

### CODE REVIEW

**Review by jharenza — CHANGES_REQUESTED** [review]
Hi @kgaonkar6! The download worked perfectly to me, but I am going to request that you add the corresponding tickets to each update noted in the `release-notes.md`

**Comment by jharenza** [comment]
Will you also add `s3://kf-openaccess-us-east-1-prd-pbta/data/release-v20-20210726/pbta-mb-pathology-subtypes.tsv` to the md5sum per #746

**Review by jharenza — APPROVED** [review]
LGTM now!

**Comment by kgaonkar6** [comment]
Please check download_script.sh again.
`release_notes.md` to be udpated once PR #1114 #1115 #1116 #1117 are merged

File: doc/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,113 @@
 # release notes
 ## current release
+### release-v20-20210726
+- release data: 2021-07-26
+- status: availale
+- changes:
+ - Update copy_number_consensus_call with manta calls with FILTER==PASS [#1113](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1113)
+    - pbta-cnv-consensus.seg.gz
```
Comment (jharenza) [inline doc/release-notes.md:8]:
```suggestion
 - Update copy_number_consensus_call with manta calls with FILTER==PASS [#1113](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1113)
 - Update reciprocal overlap rules for consensus CNV [#1125](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1125)
    - pbta-cnv-consensus.seg.gz
    - pbta-cnv-consensus-gistic.zip
```

File: doc/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,113 @@
...
+ - Update focal-cn-file-preparation with deep deletion status and with updated pbta-cnv-consensus.seg.gz above [#1099](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1099)
+    - consensus_seg_annotated_cn_autosomes.tsv.gz
+    - consensus_seg_annotated_cn_x_and_y.tsv.gz
+ - Add medulloblastoma subtypes being extracted from pathology reports [#746](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/746)
+    - pbta-mb-pathology-subtypes.tsv
+ - Add cnv and indel hostspots from MSKCC and TERT promoters [#932](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/932) 
+    - pbta-snv-scavenged-hotspots.maf.tsv.gz
+ - Update histology information 
+   - latest pathology diagnosis for 7316-2686 [#1032](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1032)
+   - BS_0TCRV9AC as sample_type == Normal
```
Comment (jharenza) [inline doc/release-notes.md:18]:
```suggestion
 - Update histology information 
   - latest pathology diagnosis for 7316-2686 [#1032](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1032)
   - BS_0TCRV9AC as sample_type == Normal
   - Add `cancer_group` for plot labeling per [#917](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/917)
```

**Review by jharenza — CHANGES_REQUESTED** [review]
few update requests on the release notes

**Comment by kgaonkar6** [comment]
Thanks for the review @jharenza !

**Comment by jaclyn-taroni** [comment]
Re-requested @jharenza to make sure all requested changes are satisfied

**Review by jharenza — APPROVED** [review]
This looks good to me! Thanks!

**Pull request merged**